src/patchRequire.js

```diff
@@ -196,10 +197,21 @@ export default function patchRequire(vol, unixifyPaths = false, Module = require
             // Don't search further if path doesn't exist
             const curPath = paths[i];
             if (curPath && stat(curPath) < 1) continue;
-            var basePath = correctPath( path.resolve(curPath, request) );
-            var filename;
-
+            var basePath = path.resolve(curPath, request);
```

Comment:
try it once without correcting the path -- if the volume supports the uncorrected path, this will allow it to work.